extensions/notebook/src/book/bookModel.ts

```diff
@@ -54,9 +54,14 @@ export class BookModel implements azdata.nb.NavigationProvider {
 
 			let p = path.join(workspacePath, '**', '_data', 'toc.yml').replace(/\\/g, '/');
 			let tableOfContentPaths = await glob(p, { deep: maxDepth });
-			this._tableOfContentPaths = this._tableOfContentPaths.concat(tableOfContentPaths);
-			let bookOpened: boolean = this._tableOfContentPaths.length > 0;
-			vscode.commands.executeCommand('setContext', 'bookOpened', bookOpened);
+			if (tableOfContentPaths.length > 0) {
+				this._tableOfContentPaths = this._tableOfContentPaths.concat(tableOfContentPaths);
+				let bookOpened: boolean = this._tableOfContentPaths.length > 0;
```

Comment:
Since you're only doing this when tableOfContentPaths.length > 0 you can just get rid of this line and set bookOpened to true directly.

Reply:
my bad, I just moved the code in if to add the error and din't take a look at what I was moving. :(
thanks for catching it.